exporter/datadogexporter/examples/collector.yaml

```
@@ -48,6 +48,25 @@ receivers:
   #       static_configs:
   #       - targets: ['0.0.0.0:8888']
 
+  # Read more about filelog receiver https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver .
+  filelog:
+    include_file_path: true
+    poll_interval: 500ms
+    include:
+     # Logs will be collected from the below file path. Please ensure it is correct or edit as needed.
```

```suggestion
      # Logs will be collected from the below file path. Please ensure it is correct or edit as needed.
```
To align with below.